Binary files pari-2.1.1/src/language/.init.c.swp and pari-2.1.1.patch/src/language/.init.c.swp differ
diff -urN pari-2.1.1/src/language/init.c pari-2.1.1.patch/src/language/init.c
--- pari-2.1.1/src/language/init.c	Mon Mar 26 16:11:43 2001
+++ pari-2.1.1.patch/src/language/init.c	Mon May 21 09:21:17 2001
@@ -1468,7 +1468,7 @@
     long delay;
 
     times(&t);
-    delay = (long)((t.tms_utime - oldticks[i]) * (1000. / CLK_TCK));
+    delay = (long)((t.tms_utime - oldticks[i]) * (1000. / CLOCKS_PER_SEC));
     oldticks[i] = t.tms_utime;
     return delay;
   }
